vector/src/main/res/values/strings.xml

```diff
@@ -591,6 +591,41 @@
     <string name="settings_app_info_link_summary">Show the application info in the system settings.</string>
 
     <string name="settings_notification_privacy">Notification privacy</string>
+    <string name="settings_notification_troubleshoot">Troubleshoot Notifications</string>
+    <string name="settings_troubleshoot_diagnostic">Troubleshoot Diagnostic</string>
+    <string name="settings_troubleshoot_diagnostic_run_button_title">Run Tests</string>
+    <string name="settings_troubleshoot_diagnostic_running_status">Running… (%1$d of %2$d)</string>
+    <string name="settings_troubleshoot_diagnostic_success_status">Basic diagnostic is ok. Please submit a bug report, we will come back to you.</string>
```

Comment:
I suggest "Basic diagnostic is ok. Please submit a bug report, to help us investigate."